pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -96,8 +96,8 @@ func setupFilterChains(authnPolicy *authn.Policy, sdsUdsPath string, sdsUseTrust
 	}
 	tls := &auth.DownstreamTlsContext{
 		CommonTlsContext: &auth.CommonTlsContext{
-			// TODO(incfly): should this be {"istio", "http1.1", "h2"}?
-			// Currently it works: when server is in permissive mode, client sidecar can send tls traffic.
+			// Note even though for PERMISSIVE mode, we match filter chain on ALPN = "istio".
```

Comment:
@PiotrSikora Is this description accurate? Would be even better if there's some envoy doc that I can link here.